CHANGES.rst

```diff
@@ -6,6 +6,7 @@ Version 0.9.2 (UNRELEASED)
 
 - Fixes ``create_workflow_from_json`` API command to always send the workflow specification to the server.
 - Fixes ``list`` command to be case-insensitive when using the ``--sort`` flag to sort the workflow runs by a specific column name.
+- Fixes ``run`` command for workflows whose specification does not contain the ``inputs`` section.
```

Comment:
Perhaps say "Fixes `run` wrapper command..." to indicate that this is just a wrapper.  Or "shortcut command" as we call it in the docs.

Also we could perhaps slightly simplify to say "... for workflows that do not contain ``inputs`` clause in their specification.

Reply:
(The "wrapper" bit will also gently highlight that "create+upload+start" was actually working well... The problem was only with the "wrapper" _per se_.)